.JuliaFormatter.toml

```diff
@@ -0,0 +1,6 @@
+always_for_in=true
+remove_extra_newlines=true
+long_to_short_function_def=true
+format_docstrings=true
+indent_submodule=true
+format_markdown=true
```

Comment:
maybe add the following format settings ?
- [`whitespace_ops_in_indices=true`](https://domluna.github.io/JuliaFormatter.jl/dev/#whitespace_ops_in_indices)
- [`whitespace_typedefs=true`](https://domluna.github.io/JuliaFormatter.jl/dev/#whitespace_typedefs)

Reply:
I don’t know if I like the white spaces. Regarding the return, yes, better to make it always true

Reply:
okay~